pages/pipelines/hosted_agents/code_access.md

```diff
@@ -1,12 +1,10 @@
 # Hosted agents code access
 
-This page details changes that need to be made to the GitHub integration to provide code access to the Buildkite hosted agents.
+Natively, Buildkite hosted agents can only be used with GitHub integrated repositories. To utilize a private repository from another provider, we've [made a plugin](https://github.com/buildkite-plugins/git-ssh-checkout-buildkite-plugin) available that's documented below.
 
 To learn more about changes that may need to be completed at an individual pipeline level, see [Pipeline migration](/docs/pipelines/hosted-agents/pipeline-migration).
 
-## Private repositories
-
-During the trial, access to private repositories is only supported through GitHub. If you have requirements to access private repositories from another source control service, please contact support.
+## GitHub private repositories
```

Comment:
step 1 of this section takes you here to the notif section

![image](https://github.com/buildkite/docs/assets/78014112/47ba16b3-c6f9-4eb7-a665-195e2bb6a6d7)

Not sure if that is what we want?
cc @mbelton-buildkite

Reply:
(not your changes @CerealBoy but thought I'd point out and confirm incase)

Reply:
@123sarahj123 This looks to be a permissions problem, as the **Repository Providers** isn't present their either (this is the same for me in prod, I can see in local where I'm admin though). I'll add a note here about having the right permissions on the Buildkite org to make the required changes.